streams/src/test/java/org/apache/kafka/streams/processor/internals/StoreChangelogReaderTest.java

```diff
@@ -443,47 +448,6 @@ public void shouldRestorePartitionsRegisteredPostInitialization() {
         assertThat(callbackTwo.restored.size(), equalTo(3));
     }
 
-    @Test
-    public void shouldThrowTaskMigratedExceptionIfEndOffsetGetsExceededDuringRestoreForChangelogTopic() {
```

Comment:
Why did we remove these tests?

Reply:
Since we cherry-picked the logic which would not throw exceptions but just retry in the next loop, these tests are not needed any more -- in 2.0 they are removed as well.

Reply:
Got it, thanks for the explanation.

Reply:
Ah got it, thanks for the explanation.